src/hooks/use-photo-filter.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { renderHook } from '@testing-library/react';
+import { usePhotoFilter } from './use-photo-filter';
+import { QueryProvider } from 'context/';
+
+const testName = 'testName';
+const testValue = 'testValue';
+
+describe('usePhotoFilter', () => {
+  test('should work correct', () => {
+    const { result } = renderHook(usePhotoFilter, { wrapper: QueryProvider });
+    expect(typeof result.current).toBe('function');
```

Comment:
really?)

u should test updating data in cache instead of testing returned value

Reply:
i have rewrited test for new hook